action.yml

```
@@ -29,31 +84,65 @@
+    # and optionally download missing Apple platforms.
+    
+    # Extract Xcode version name from the provided Xcode path
+    # Example: '/Applications/Xcode_15.1.app' → 'XCODE_NAME=15.1'
+    # This version identifier is used for cache key generation to ensure
+    # build artifacts are isolated between different Xcode versions
     - name: Set Xcode Name
       if: steps.detect-os.outputs.os == 'macos' && inputs.xcode
       shell: bash
       working-directory: ${{ inputs.working-directory }}
       run: echo "XCODE_NAME=$(basename -- ${{ inputs.xcode }} | sed 's/\.[^.]*$//' | cut -d'_' -f2)" >> $GITHUB_ENV
     
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)